### PR TITLE
[IR] Bluray/DVD Universal Remote

### PR DESCRIPTION
**Pull request by jaylikesbunda**
# What's New

### NEW Remote Icons:
_Eject
CC (Closed Captions/Subtitle)
Ok
Fast F
Fast B_

### New `bluray_dvd.ir` file with:

| Function       | Signals |
|----------------|---------|
| Power          | 40      |
| Eject          | 24      |
| Play           | 35      |
| Pause          | 28      |
| Fast Forward   | 17      |
| Fast Backward  |  26     |
| Ok          | 27      |
| CC             | 20      |

Screenshot-20241008-222309

big thanks to @luu176 and @Willy-JL for the help couldn't have made this without them

-----
# For the reviewer

- [x] I've uploaded the firmware with this patch to a device and verified its functionality
- [x] I've confirmed the bug to be fixed / feature to be stable

### CODE REVIEW

File: targets/f7/api_symbols.csv
Comment (jaylikesbunda) [inline targets/f7/api_symbols.csv:1]: what

**Comment by jaylikesbunda** [comment]
fbt automatically bumped f7 api_symbols to 73.2 but i manaually bumped f18 to 73.2 to prevent build check error

**Comment by WillyJL** [comment]
> fbt automatically bumped f7 api_symbols to 73.2 but i manaually bumped f18 to 73.2 to prevent build check error

yep FBT does that, but since we're not in charge here we must follow OFW API versioning or apps wont work, so we can never update API version independently... i should just disable it automatically changing API version in our FBT

**Comment by WillyJL** [comment]
> ### TODO
> * rework and re-evaluate buttons

they look reasonable to me, only note is that closed captions could use being 2 pixel wide stroke instead of 1. do you also plan to change which buttons are included, like including different functions?

**Comment by WillyJL** [comment]
as far as im concerned this is good to merge, but heard on discord you might want to swap the STOP button. whatever you choose, undraft when ready :D

**Comment by jaylikesbunda** [comment]
> i'm sorry. i genuinely cannot build this, everytime i go to it tells me i'm using fast_f instead of "fast_foward" which was a misspelling and is not listed anywhere and has been renamed everywhere. i have no idea what's going on. everything is done and ready

never give up or something idk

**Comment by WillyJL** [comment]
> i'm sorry. i genuinely cannot build this, everytime i go to it tells me i'm using fast_f instead of "fast_foward" which was a misspelling and is not listed anywhere and has been renamed everywhere. i have no idea what's going on. everything is done and ready

probably due to how i updated your branch. im assuming you added more changes after i had edited the branch, but you had not pulled my changes first. if everything is correct on github, but your local clone is broken, you can try to delete the local branch and clone the remote branch again.

```bash
git branch -m temp
git checkout -b dev origin/dev
git branch -D temp
```
this will rename current branch, checkout the remote branch (you might have to change origin based on your configured remote name), then delete the old branch

File: applications/main/infrared/infrared_scene_universal_bluray.c [hunk 1]
```diff
@@ -0,0 +1,169 @@
+#include "../infrared_app_i.h"
```
Comment (WillyJL) [inline applications/main/infrared/infrared_scene_universal_bluray.c]: looks like you uploaded this in the wrong place )

Comment (jaylikesbunda) [reply applications/main/infrared/infrared_scene_universal_bluray.c]: fixing everything now i figured it out :)

**Review by WillyJL — APPROVED** [review]
LGTM, nice!

**Pull request merged**